CMakeLists.txt

```diff
@@ -117,6 +117,8 @@ set(GENCODE_SM37
   -gencode=arch=compute_37,code=sm_37 -gencode=arch=compute_37,code=compute_37)
 set(GENCODE_SM50
   -gencode=arch=compute_50,code=sm_50 -gencode=arch=compute_50,code=compute_50)
+set(GENCODE_SM52
```

Comment:
Includes the Maxwell GPUs (GTX 970, etc.) that we use in lab for our personal computers and are also widely available.